<!-- -->
<!-- Convert the depth image and the rgb image into a xyzrgb Point Cloud -->
<launch>

    <node name="update_frame_id" pkg="depth_image_proc" type="update_frame_id.py" output="screen" />

    <!-- Nodelet manager for this pipeline -->
    <node pkg="nodelet" type="nodelet" args="manager" name="standalone_nodelet" output="screen"/>


    <!-- Convert to point cloud -->
    <node pkg="nodelet" type="nodelet" name="depth_image_proc" args="load depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
   
        <!--remap from your specific sensors-->         

        <!-- Input: Camera calibration and metadata.  (sensor_msgs/CameraInfo) -->
        <!--remap from="rgb/camera_info" to="/carla/ego_vehicle/rgb_front/camera_info"/-->

        <!-- Input: Rectified color image. (sensor_msgs/Image) -->
        <!--remap from="rgb/image_rect_color" to="/carla/ego_vehicle/rgb_front/image"/-->

        <!-- Input: Rectified depth image, registered to the RGB camera. (sensor_msgs/Image) -->
        <remap from="depth_registered/image_rect" to="/carla/ego_vehicle/depth_front/image"/>

        <!-- Output: XYZ point cloud. If using PCL, subscribe as PointCloud<PointXYZ>.  (sensor_msgs/PointCloud2)  -->
        <remap from="depth_registered/points" to="/point_cloud/points"/>

    </node>

</launch>

<!-- -->
